ext/pgsql/tests/pg_fetch_object_with_abstract_class.phpt

```diff
@@ -0,0 +1,58 @@
+--TEST--
+pg_fetch_object() with abstract class name
+--EXTENSIONS--
+pgsql
+--SKIPIF--
+<?php
+include("skipif.inc");
+?>
+--FILE--
+<?php
+
+interface I {}
+
+abstract class C {}
+
+enum E {
+	case A;
+}
+
+include "config.inc";
+$table_name = "pg_fetch_object_abstract_class";
+$db = pg_connect($conn_str);
+pg_query($db, "CREATE TABLE {$table_name} (a integer, b text)");
+pg_query($db, "INSERT INTO {$table_name} VALUES(0, 'ABC')");
+
+$sql = "SELECT * FROM $table_name WHERE a = 0";
+
+try {
+	$result = pg_query($db, $sql);
+	var_dump(pg_fetch_object($result, NULL, 'I'));
+} catch(Throwable $e) {
+	echo $e::class, ': ', $e->getMessage(), PHP_EOL;
+}
+
+try {
+	$result = pg_query($db, $sql);
+	var_dump(pg_fetch_object($result, NULL, 'C'));
+} catch(Throwable $e) {
+	echo $e::class, ': ', $e->getMessage(), PHP_EOL;
+}
+
+try {
+	$result = pg_query($db, $sql);
+	var_dump(pg_fetch_object($result, NULL, 'E'));
+} catch(Throwable $e) {
+	echo $e::class, ': ', $e->getMessage(), PHP_EOL;
+}
+
```

Comment:
nit: would it be possible to add the closing tag ?

Reply:
Might also be the reason windows is confused